app/views/users/show.html.erb

```diff
@@ -8,36 +8,70 @@
   <%= button_to 'Log Out', logout_path, method: 'delete', class: "btn btn-primary mb1 bg-teal" %>
   <h3>Account Details</h3>
   <ul>
-    <li> <%= current_user.first_name  %> <%= current_user.last_name %> </li>
-    <li> <%= current_user.email%> </li>
+    <li>
+      <%= current_user.first_name  %> <%= current_user.last_name %>
```

Comment:
Does having a combined .name stored in the DB make sense or overkill? Seems like that falls under DRY principals...